### ISSUE TITLE
Taxonomy Tiles for all post types

### ISSUE THREAD

**Issue by scubaninja1977** [CrocoBlock/suggestions#301]
Add the ability to create Taxonomy Tiles to all post types.

Currently, this is only available for Woocommerce products.

This would be nice to have so when we are showcasing taxonomies on the home page for example, like you can currently do for products.

**Comment by Floydtm** [comment]
@scubaninja1977 
Hi, You can use the Smart Posts Tiles widget looking for a plugin jet blog. In it you can specify the type of post http://prntscr.com/nxdrp2 http://prntscr.com/nxdsx9

**Comment by scubaninja1977** [comment]
Hi, I have tested this and it's not working, allow me to explain a little further.

I have custom taxonomy for Diving Locations, I added a custom field to add an image to that taxonomy that is defined.

So basically I need to display the taxonomy tiles and not the post types as its not a post type that is set up its an actual taxonomy.

Jet woo builder has actual taxonomy tiles that work in this manner and not the way you describing it, so basically as i said, i need a way to use the taxonomy tiles but for custom taxonomies.

thank you. :)
